openmrs-19/src/main/java/org/motechproject/openmrs19/resource/impl/EncounterResourceImpl.java

```diff
@@ -80,4 +92,16 @@ private Gson buildGsonWithAdapters() {
                 .registerTypeAdapter(Observation.class, new Observation.ObservationSerializer())
                 .create();
     }
+
+    /**
+     * Rewrites provider from the list to single provider object in encounter. Since OpenMRS version 1.10
```

Comment:
Sets provider from the list to a single provider object in encounter. ... as a list